aws/resource_aws_qldb_stream_test.go

```diff
@@ -0,0 +1,322 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/qldb"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_qldb_stream", &resource.Sweeper{
+		Name: "aws_qldb_stream",
+		F:    testSweepQLDBStreams,
+	})
+	testAccProviders = map[string]*schema.Provider{
+		ProviderNameAws: testAccProvider,
+		// "time":          testAccProvider,
```

Comment:
This is what is missing to get this test passing 😬 
I haven't been able to get the full set of resources needed to set up a stream running without delaying to let the `aws_iam_role`'s policy to take effect before creating the `aws_qldb_stream`